### DISCUSSION TITLE
Editing rows with multi-select and checkbox fields

### DISCUSSION THREAD

**Discussion by stavrianosy** [KevinVandy/material-react-table#412] [Q&A]
I am using the editing feature and I need to include a checkbox and a multi-select fields.
Can you provide and example on how to implement this.
I am following the example from here https://www.material-react-table.com/docs/examples/editing-crud but there is no mention for the checkbox and the multi-select.

Thanks

**Comment by VeenaGangal** [comment]
@stavrianosy- I have implemented same functionality. You can pass MUI multiselect component as Edit prop to coulmn.

Attaching sample code for your reference.
[customEditComponent.txt](https://github.com/KevinVandy/material-react-table/files/12037910/customEditComponent.txt)
